app/views/overview/_service-bindings.html

```diff
@@ -19,7 +19,7 @@
   <p ng-if="($ctrl.refApiObject.kind !== 'ServiceInstance')  && !($ctrl.bindableServiceInstances | size)">
     <span>You must have a bindable service in your namespace in order to create bindings.</span>
     <div>
-      <a href="./">Browse Catalog</a>
+      <a href="./browse-catalog">Browse Catalog</a>
```

Comment:
```html
<a ng-href="{{project | catalogURL}}">
```

Reply:
Hmm...I don't see `project` defined in `/app/scripts/directives/overview/serviceBindings.js`